webapp/channels/src/components/channel_header/__snapshots__/channel_header.test.tsx.snap

```diff
@@ -4700,7 +4685,6 @@ exports[`components/ChannelHeader should render the pinned icon with the pinned
                     </span>
                   </strong>
                   <span
-                    aria-label="dropdown icon"
```

Comment:
This fixes the issue in the screenshot. Such ARIA label is redundant to the parent (button) ARIA label as `channel menu` which acts as single interactive element. Dropdown icon is not accessible/interactive by its own alone. 
<img width="588" alt="Screenshot 2023-07-26 at 4 45 44 PM" src="https://github.com/mattermost/mattermost/assets/5334504/72d37623-f6d5-4709-98ca-c6a73f58fa5b">